src/set_theory/cardinal.lean

```diff
@@ -145,10 +145,13 @@ map ulift (λ α β e, equiv.ulift.trans $ e.trans equiv.ulift.symm) c
 theorem lift_umax : lift.{(max u v) u} = lift.{v u} :=
 funext $ λ a, induction_on a $ λ α, (equiv.ulift.trans equiv.ulift.symm).cardinal_eq
 
+theorem lift_umax' : lift.{(max v u) u} = lift.{v u} := lift_umax
```

Comment:
should this one and `lift_umax` be simp lemmas?

Reply:
I tried, and it misfired at some proofs. I can try it again, but in another PR.